pkg/lib/_global-variables.scss

```diff
@@ -3,3 +3,16 @@
  */
 $grid-gutter-width: 30px;
 $font-size-base: 1;
+
+/*
+ * PatternFly 4 adapting the lists too early.
+ * When PF4 has a breakpoint of 768px width, it's actually 1108 for us, as the sidebar is 340px.
+ * (This does use the intended content area, but there's a mismatch between content and browser width as we use iframes.)
+ * So redefine grid breakpoints
+ */
```

Comment:
I guess the purpose of this  is to adjust PF variables everywhere? Is that a magic file name, or how is it actually being used? It's not mentioned anywhere except in a comment in webpack.config.js.

Reply:
It's made visible everywhere in cockpit through https://github.com/cockpit-project/cockpit/blob/master/webpack.config.js#L439

And exactly - it's adjusting this variables everywhere.